src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -337,8 +334,15 @@ void HexagonGame::updateRotation(FT mFT)
             getSign(nextRotation);
         status.fastSpin -= mFT;
     }
-
-    backgroundCamera.turn(nextRotation);
+    levelStatus.rotation += nextRotation;
+    if(levelStatus.rotation >= 360.f)
```

Comment:
Probably won't happen, but what if `nextRotation` is greater than 360?

Reply:
Will cap it then with modulo, maybe, will try to figure it out actually